lib/fennec/stun/auth.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Fennec.STUN.Auth do
+  @moduledoc """
+  This module implements authentication and authorization of the STUN protocol.
```

Comment:
This module is probably not to be exposed to end users, so it should be `@moduledoc false`.